src/main/java/uk/gov/crowncommercial/dts/scale/cat/config/JaggaerOAuth2Config.java

```diff
@@ -96,10 +100,15 @@ public Request newRequest(URI uri) {
     };
     ClientHttpConnector jettyHttpClientConnector = new JettyClientHttpConnector(client);
 
+    // SCC-517
+    var contentTypeHeaderFilter = ExchangeFilterFunction.ofResponseProcessor(clientResponse -> Mono
+        .just(clientResponse.mutate().headers(headers -> headers.remove(HttpHeaders.CONTENT_TYPE))
+            .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE).build()));
```

Comment:
👍

Reply:
Ha oops - this change was meant to be on a new branch.  I might move it across just to keep them separate..

Reply:
😄 I had just assumed you were rolling it into this PR - whichever you prefer